mappings/remind_2.1.yaml

```diff
@@ -22,8 +22,9 @@ native_regions:
   - SSA: REMIND 2.1|SSA
   - UKI: REMIND 2.1|UKI
   - USA: REMIND 2.1|USA
-  - EUR: EU27 & UK
-  - World
+  - EU27 & UK: REMIND 2.1|EUR
```

Comment:
I must confess I did not understood 100% the mapping files. Could you point me to a documentation or explain me what is the meaning of what is before and after the colon in both files: mappings/remind_2.1.yaml and definitions/region/model_native_regions/remind_2.1.yaml 

 - `EUR` is the native region in REMIND and this should be renamed to `EU27 & UK` in the iiasa database.
 - I am not using automated aggregates for the EUR region because the need to include carbon price and other prices in the results, and as far as I understood the iiasa database does not support weights for the prices aggregations, am I wrong?

Reply:
See the documentation of the model-region-mapping files [here](https://nomenclature-iamc.readthedocs.io/en/latest/usage.html#regionprocessor).

pyam (and therefore the database processing infrastructure) does support weighted average, but there was no decision about prices should be handled in the ECEMF project (see my email on Nov 19 and following emails).